### PR TITLE
Feature/mb tstep to double

### PR DESCRIPTION
**Pull request by Bellout**
Changed TSTEP type from int to double.
Changed absolute "/home/Einar/..." to relative paths -> all tests pass now.
Some format changes (Google style).

### CODE REVIEW

File: FieldOpt/Simulation/simulator_interfaces/driver_file_writers/driver_parts/ecl_driver_parts/runspec_section.cpp [hunk 1]
```diff
@@ -42,7 +42,8 @@ Runspec::Runspec(QStringList *driver_file_contents, QList<Model::Wells::Well *>
 
     // Create proper WELLDIMS keyword
     int maxwells = wells->size();
-    int maxconns = wells->first()->trajectory()->GetWellBlocks()->size() * (maxwells +1);
+//    int maxconns = wells->first()->trajectory()->GetWellBlocks()->size() * (maxwells +1);
+    int maxconns = 300;
```
Comment (einar90) [inline FieldOpt/Simulation/simulator_interfaces/driver_file_writers/driver_parts/ecl_driver_parts/runspec_section.cpp]: Why this change? Could it not cause problems?

Comment (Bellout) [reply FieldOpt/Simulation/simulator_interfaces/driver_file_writers/driver_parts/ecl_driver_parts/runspec_section.cpp]: No problems. Old maxconns is sometimes too small, thus stopping the whole sim. Sometimes opt starts with really small wells that develop into very large ones.

File: FieldOpt/Simulation/simulator_interfaces/driver_file_writers/driver_parts/ecl_driver_parts/welspecs.cpp [hunk 2]
```diff
@@ -2,63 +2,65 @@
...
-                        foot_ = "/\n\n";
-                        for (int i = 0; i < wells->size(); ++i) {
-                            if (wells->at(i)->trajectory()->GetWellBlocks()->size() > 0)
-                                entries_.append(createWellEntry(wells->at(i)));
-                        }
-                    }
+Welspecs::Welspecs(QList<Model::Wells::Well *> *wells)
+{
+    initializeBaseEntryLine(10);
+    head_ = "RPTRST\n'BASIC=3'\n/\n\nWELSPECS";
```
Comment (einar90) [inline FieldOpt/Simulation/simulator_interfaces/driver_file_writers/driver_parts/ecl_driver_parts/welspecs.cpp]: I don't agree with this change. Writing this output slows down the simulation, and is of no use to the optimization. It should not be forced in by FieldOpt. Better to add it to the deck elsewhere (in the parts not touched by FieldOpt), or just add it after the optimization is done.

Comment (Bellout) [reply FieldOpt/Simulation/simulator_interfaces/driver_file_writers/driver_parts/ecl_driver_parts/welspecs.cpp]: Really? Never noticed. Is it really that much compared to everything else? If this is in, then one is able to view the performance of the  candidate solutions during runtime. The keywords needs to be added after SCHEDULE. If you are hard set against it, just revert the change, and merge.

**Comment by Bellout** [comment]
Reverted latest commit that added RPTRST to WELSPEC. Can this be merged now?

**Comment by einar90** [comment]
It can, sorry for the delay!
